LOAN/LoanTypes/LoanProducts.rdf

```diff
@@ -103,30 +123,163 @@
+		<rdf:type rdf:resource="&fibo-loan-typ-prod;LoanMarketCategory"/>
+		<rdfs:label>small business loan</rdfs:label>
+		<fibo-fnd-utl-av:explanatoryNote>This is related to the mortgage loan purpose: &apos;business or commercial&apos;.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:NamedIndividual>
+	
+	<owl:Class rdf:about="&fibo-loan-typ-prod;LoanProduct">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-pas-fpas;FinancialProduct"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fbc-pas-fpas;isEmbodiedIn"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-ln;Loan"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;refersTo"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-dae-dbt;Collateral"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-loan-typ-prod;hasPreconditions"/>
+				<owl:someValuesFrom rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/DebtAndEquities/CreditFacilities/LoanPrecedentConditions"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-loan-typ-prod;isTemplateFor"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-ln;Loan"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>loan product</rdfs:label>
+		<skos:definition>financial product that is realized as a loan that a borrower may acquire from a lending institution with specific characteristics and terms</skos:definition>
+	</owl:Class>
+	
 	<owl:Class rdf:about="&fibo-loan-typ-prod;LoanProductDisclosureRight">
 		<rdfs:subClassOf rdf:resource="&fibo-loan-loan-reg;ProductDisclosureRight"/>
 		<rdfs:label xml:lang="en">loan product disclosure right</rdfs:label>
 		<skos:definition xml:lang="en">Right to fair representation of the facts about Loan products before entrering into agreements for same.</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-loan-typ-prod;LoanProductOffering">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-pas-fpas;Offering"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-lr;has"/>
+				<owl:onClass rdf:resource="&fibo-loan-typ-prod;LineItem"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;appliesTo"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-typ-prod;LoanProduct"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>loan offering</rdfs:label>
+		<skos:definition>generic loan product offered to the public, or a specific offer tailored to particular circumstances aimed at one potential borrower</skos:definition>
```

Comment:
what's the point of LoanProduct that is NOT offered to the public?

Reply:
No idea.  I'll go back in and give this a better definition and fix the XML entity issue, but not changing anything else under this pull request, which was mainly to fix LoanCore.

Reply:
I couldn't find a specific definition online or in Barrons, so I've made it better but a working group can likely improve on it.